app/views/spree/shared/_reviews.html.erb

```diff
@@ -1,12 +1,8 @@
 <div id="reviews">
-  <h3 class="product-section-title"><%= Spree.t(:reviews) %></h3>
   <% if Spree::Reviews::Config[:include_unapproved_reviews] == false and @product.reviews.approved.count == 0 %>
-    <p class="reviews-none"><%= Spree.t(:no_reviews_available) %></p>
+  	<%= render 'spree/reviews/stars', stars: 0 %>
   <% else %>
     <%= render 'spree/shared/rating', product: @product, review: 0 %>
-    <% for review in (Spree::Reviews::Config[:track_locale] ? @product.reviews.localized(I18n.locale) : @product.reviews).default_approval_filter.preview %>
```

Comment:
Why you remove this LOOP. What is the purpose of that? Need your explanation

Reply:
its added in partial